newsfragments/8272.changed

```diff
@@ -0,0 +1 @@
+Added towncrier for automatic changelog generation
```

Comment:
i personally don't think our users need to hear about this and would recommend deleting it, but you can keep it if you want to